TF_Settings_Web/src/Pages/Visuals/VisualsScreen.tsx

```diff
@@ -47,11 +47,8 @@ const bgImages = [GradientBg, WhiteTextBg, BlackTextBg, MountainBg];
 const bgPreviewImages = [GradientBgPreview, WhiteTextBgPreview, BlackTextBgPreview, MountainBgPreview];
 const closeCtiOptions = ['Users Hand Present', 'User Performs Interaction'];
 
-const reducer = (state: VisualsConfig, payload: { content: Partial<VisualsConfig>; writeOutConfig: boolean }) => {
+const reducer = (state: VisualsConfig, payload: { content: Partial<VisualsConfig> }) => {
```

Comment:
:eyes: You can take this out of payload now and just use `content: Partial<VisualsConfig>` - the payload wrapper was needed when there was multiple inputs (i.e. content and writeOutConfig) as the reducer only takes one (+ a parameter for the current state)